src/libecalc/common/graph.py

```diff
@@ -49,6 +49,15 @@ def get_successors(self, node_id: NodeID, recursively=False) -> List[NodeID]:
         else:
             return list(self.graph.successors(node_id))
 
+    def get_successor(self, node_id: NodeID) -> NodeID:
```

Comment:
Current limitation that we want to have now, or always, to use "crossovers" when there are more than 1?

Reply:
Not always, currently it's used when it's supposed to only be one, but we could of course just use get_successors()[0]. I was thinking we could have both get_successor and get_successors. I think we already have both for either successors or predecessors.